<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>87</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonOpen">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="buttonGaussien">
       <property name="text">
        <string>Gaussien</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="buttonMedian">
       <property name="text">
        <string>Median</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>11</y>
      <width>911</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuFlou">
    <property name="title">
     <string>Filtres</string>
    </property>
    <widget class="QMenu" name="menuGaussien">
     <property name="title">
      <string>Gaussien</string>
     </property>
     <addaction name="actionLeger"/>
     <addaction name="actionMoyen"/>
     <addaction name="actionFort"/>
    </widget>
    <widget class="QMenu" name="menuGradient">
     <property name="title">
      <string>Gradient</string>
     </property>
     <addaction name="actionX"/>
     <addaction name="actionY"/>
    </widget>
    <addaction name="menuGaussien"/>
    <addaction name="actionMedian"/>
    <addaction name="actionPasse_Haut"/>
    <addaction name="actionRehaussment_Contours"/>
    <addaction name="menuGradient"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuFlou"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>Median</string>
   </property>
  </action>
  <action name="actionLeger">
   <property name="text">
    <string>Leger</string>
   </property>
  </action>
  <action name="actionMoyen">
   <property name="text">
    <string>Moyen</string>
   </property>
  </action>
  <action name="actionFort">
   <property name="text">
    <string>Fort</string>
   </property>
  </action>
  <action name="actionPasse_Haut">
   <property name="text">
    <string>Passe Haut</string>
   </property>
  </action>
  <action name="actionRehaussment_Contours">
   <property name="text">
    <string>Rehaussement Contours</string>
   </property>
  </action>
  <action name="actionX">
   <property name="text">
    <string>X</string>
   </property>
  </action>
  <action name="actionY">
   <property name="text">
    <string>Y</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
